docs/production-deployment/self-hosted-guide/temporal-nexus.mdx

```diff
@@ -57,3 +69,28 @@ To enable Nexus in your deployment:
          - Pattern: "*"
            AllowInsecure: true
    ```
+
+## Build and use Nexus Services
+
+Nexus has a familiar programming model to build and use Nexus Services using the Temporal SDK.
+The [Nexus Operation lifecycle](/nexus/operations#operation-lifecycle) supports both synchronous and asynchronous Operations.
+Nexus Operations can be implemented with Temporal primitives, like a Workflow, or execute arbitrary code.
+
+:::tip RESOURCES
+
+- [Go SDK - Nexus quick start and code sample](/develop/go/nexus)
+- [Java SDK - Nexus quick start and code sample](/develop/java/nexus)
+
+:::
+
+## Learn more
+
+Evaluate if you should use Nexus and learn how it works.
+
+:::tip RELATED
+
+- [Evaluate](/evaluate/nexus) why you should use Nexus and watch the [Nexus keynote and demo](https://youtu.be/qqc2vsv1mrU?feature=shared&t=2082).
+- Learn how Nexus works in the [Nexus deep dive talk](https://www.youtube.com/watch?v=izR9dQ_eIe4) and [Encyclopedia](/nexus).
```

Comment:
Should we link directly to the how it works section? https://www.youtube.com/watch?v=izR9dQ_eIe4&t=934s

Reply:
yup, fixed